Toolchain/Patches/llvm/0003-Driver-Add-support-for-SerenityOS.patch

```diff
@@ -224,6 +227,15 @@ index 000000000..4d653d86f
 +    CmdArgs.push_back(Output.getFilename());
 +  }
 +
++  const char *Exec = Args.MakeArgString(TC.GetLinkerPath());
++  auto linkerIs = [Exec](const char* name) {
++    return llvm::sys::path::filename(Exec).equals_insensitive("ld.lld") ||
```

Comment:
Should this pass `name` instead of "ld.lld"? Looks like we won't pack relocs with mold

Reply:
🤦‍♂️